spec/services/claims/school/accept_grant_conditions_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+describe Claims::School::AcceptGrantConditions do
+  subject(:grant_conditions) { described_class.call(school:, user:) }
+
+  let(:school) { create(:claims_school, :claims, urn: "123457", claims_grant_conditions_accepted_at: nil) }
+
+  let(:user) do
+    create(
+      :claims_user,
+      :anne,
+      user_memberships: [create(:user_membership, organisation: school)],
+    )
+  end
+
+  before do
+    Timecop.freeze(Time.zone.local(2024, 4, 25, 12, 0, 0))
```

Comment:
Also personal preference, feel free to ignore.

This takes some time to parse as a human reading it. Using `Time.zone.parse` helps with readability of the specs.

```suggestion
    Timecop.freeze(Time.zone.parse("25 April 2024 12:00"))
```